### PR TITLE
feat(forum): add forum page

### PR DESCRIPTION
**Pull request by BilalShandyartaSyamsudin**
Check Proggress #4

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -29,6 +30,11 @@ const router = createBrowserRouter(
   createRoutesFromElements(
     <Route path="/" element={<Root />} errorElement={<Error />}>
       <Route index element={<Home />} />
+
+      <Route path="/about" element={<About />} />
+      <Route path="/login" element={<Login />} />
+      <Route path="/register" element={<Register />} />
```
Comment (tokimina) [inline src/main.tsx]: Double Cuy hapus

**Review by tokimina — CHANGES_REQUESTED** [review]

File: src/main.tsx (hunk 1 above)
Comment (BilalShandyartaSyamsudin) [reply src/main.tsx]: done

Comment (tokimina) [reply src/main.tsx]: about e drung

Comment (BilalShandyartaSyamsudin) [reply src/main.tsx]: wes cek

**Pull request closed without merging**